seatunnel-transforms-v2/src/main/java/org/apache/seatunnel/transform/sql/zeta/ZetaSQLFunction.java

```diff
@@ -193,6 +207,16 @@ public Object computeForValue(Expression expression, Object[] inputFields) {
         if (expression instanceof NullValue) {
             return null;
         }
+        if (expression instanceof TrimFunction) {
```

Comment:
what's for?

Reply:
sql trim() will parse to TrimFunction ,and it is not extends Function.